ReactWindows/ReactNative/Views/Picker/ReactPickerManager.cs

```diff
@@ -0,0 +1,210 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Events;
+using System;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace ReactNative.Views.Picker
+{
+    /// <summary>
+    /// A view manager responsible for rendering picker.
+    /// </summary>
+    public class ReactPickerManager : BaseViewManager<ComboBox, ReactPickerShadowNode>
+    {
+        private const string PROP_COLOR     = "color";
+        private const string PROP_ENABLED   = "enabled";
+        private const string PROP_ITEMS     = "items";
+        private const string PROP_LABEL     = "label";
+        private const string PROP_SELECTED  = "selected";     
+
+        private int _selected;
+
+        /// <summary>
+        /// The name of the view manager.
+        /// </summary>
+        public override string Name
+        {
+            get
+            {
+                return "RCTPicker";
+            }
+        }
+
+        /// <summary>
+        /// Sets whether a picker is enabled.
+        /// </summary>
+        /// <param name="view">a combobox view.</param>
+        /// <param name="enabled">
+        /// Set to <code>true</code> if the picker should be enabled,
+        /// otherwise, set to <code>false</code>.
+        /// </param>
+        [ReactProperty(PROP_ENABLED)]
+        public void SetEnabled(ComboBox view, bool enabled)
+        {
+            view.IsEnabled = enabled;
+        }
+
+        /// <summary>
+        /// Sets the selected item.
+        /// </summary>
+        /// <param name="view">a combobox instance.</param>
+        /// <param name="selected">The selected item.</param>
+        [ReactProperty(PROP_SELECTED)]
+        public void SetSelected(ComboBox view, int selected)
+        {
+            // Temporarily disable selection changed event handler.
+            view.SelectionChanged -= OnSelectionChanged;
+
+            _selected = selected >= -1 ? selected : -1;
+            view.SelectedIndex = view.Items.Count > _selected ? _selected : view.Items.Count - 1;

```

Comment:
Is this correct behavior to choose the last item (I honestly don't know, I'm not saying it's wrong, just asking), or should we throw an exception here?